src/commands/blocks.ts

```
@@ -482,4 +483,4 @@ export const blocksCommand = define({
 			}
 		}
 	},
-});
+});
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)